QEfficient/utils/device_utils.py

```diff
@@ -27,3 +28,41 @@ def get_available_device_id():
             elif "Failed to find requested device ID" in result.stdout:
                 print("Failed to find requested device ID")
                 return None
+            
+def is_qpc_size_gt_32gb(params: int, mxfp6: bool) -> bool:
+    if mxfp6:
+        qpc_size = math.ceil((params * 1) / Constants.GB)
+    else:
+        qpc_size = math.ceil((params * 2) / Constants.GB)
+
+    print(f"Approximate QPC size is: {qpc_size} GB")

```

Comment:
Should we use `logger.info` ?

Reply:
Right now, it is not getting dumped to log file. So will make the change post that. This is for user to know.